recipes/keras-resnet/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "keras-resnet" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "8ce27ba782d1b45b127af51208aefdceb2de8d2c54646bac5fc786506ce558c0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [unix]
+    - "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - python
+    - pip
```

Comment:
shouldn't you add ``setuptools_scm`` as well?
I didn't check but could you please confirm? Because you are using the ``SETUPTOOLS_SCM_PRETEND_VERSION``